web/src/app/projects/repository/repository.component.scss

```diff
@@ -6,3 +6,77 @@ mat-list {
   overflow: hidden;
   text-overflow: ellipsis;
 }
+
+.dashboard-card.mat-card-content{
```

Comment:
I think the formatting in this file is not consistent, for example, I think this line should be...

```css
.dashboard-card .mat-card-content {
```

Reply:
This is not formatting . This is selector :)

Reply:
Some places have a space between the selector and the opening bracket, which I think reads better

Reply:
Sure !! I will update